frontend/src/concepts/areas/types.ts

```diff
@@ -58,6 +61,7 @@ export enum StackComponent {
   WORKBENCHES = 'workbenches',
   TRUSTY_AI = 'trustyai',
   KUEUE = 'kueue',
+  MODEL_REGISTRY = 'trustyai', // TODO: Change to model-registry once the component is available in DSC
```

Comment:
This is just a workaround in place, if not it's gonna be really hard to test the feature until is enabled by the operator.

cc @zdtsw any idea on how to hack a cluster to show an unavailable dsc component?